upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -749,3 +753,113 @@ func (c *awsCloudImplementation) FindVPCInfo(vpcID string) (*fi.VPCInfo, error)
 
 	return vpcInfo, nil
 }
+
+// DefaultInstanceType determines an instance type for the specified cluster & instance group
+func (c *awsCloudImplementation) DefaultInstanceType(cluster *kops.Cluster, ig *kops.InstanceGroup) (string, error) {
+	var candidates []string
+
+	switch ig.Spec.Role {
+	case kops.InstanceGroupRoleMaster:
+		// Some regions do not (currently) support the m3 family; the c4 large is the cheapest non-burstable instance
```

Comment:
Why can we change now... ? If we really can't change now- maybe we start by throwing an errata in the logs that lets people know that we are phasing %s instance out in x version.